clients/src/main/java/org/apache/kafka/common/network/Selector.java

```diff
@@ -78,6 +78,7 @@
 public class Selector implements Selectable {
 
     public static final long NO_IDLE_TIMEOUT_MS = -1;
+    private static final long CLOSE_TIMEOUT_MS = 60000;
```

Comment:
Rather than hard-coding the timeout here, would it make sense to add the timeout parameter to `closeGracefully`?